models/cardHistory.js

```diff
@@ -58,11 +57,6 @@ const cardHistorySchema = mongoose.Schema({
   },
 });
 
-cardHistorySchema.index({ oracleId: 1, 'current.rating': 1 });
-cardHistorySchema.index({ oracleId: 1, 'current.elo': -1 });
-cardHistorySchema.index({ oracleId: 1, 'current.picks': -1 });
-cardHistorySchema.index({ oracleId: 1, 'current.cubes': -1 });
```

Comment:
Just flagging this to mention that this change won't (I believe) remove the already built indexes from the database - those have to be deleted manually.

Reply:
This is correct, this is something I handle manually